src/app/public/src/modules/link/link.component.html

```diff
@@ -0,0 +1 @@
+<a class="stache-link" (click)="navigate()"><ng-content></ng-content></a>
```

Comment:
With this being an anchor link do we have to do anything with stopping event propagation from the default anchor link click behavior?

Reply:
We don't do that anyway with nav links. But I can swap it to a div.

Reply:
I used an anchor tag so I wouldn't have to recreate the styles.

Reply:
https://github.com/blackbaud/stache2/blob/master/src/app/public/src/modules/nav/nav.component.ts#L35
https://github.com/blackbaud/stache2/blob/master/src/app/public/src/modules/nav/nav.component.html#L6

Reply:
I think we're fine, especially since we don't have the href on there.

Reply:
I definitely like the anchor link :) It was an actual question lol I don't know if there would be bubbling problems or not.

Reply:
👍